docs/security/host-certs.md

```diff
@@ -152,7 +155,7 @@ Once you have RA privileges, you may request, approve, and retrieve host certifi
         user@host $ osg-incommoncert-request --username <INCOMMON_LOGIN> \
                     --cert ~/path_to_dir/incommon_user_cert.pem \
                     --pkey ~/path_to_dir/incommon_user_key.pem \ 
-                    --hostname <HOSTNAME>
+                    --hostname <HOSTNAME> [--orgcode <ORG,DEPT>]
```

Comment:
Could we put the `--orgcode` on a newline to mimic the formatting of the other options?